docs/scripting/language/Style.md

```diff
@@ -221,22 +221,29 @@ Enumerators, if named, must be prefixed with `E_` and use
 `Index tag mismatch` warning, in which case an `e_` prefix may be used.
 `SCREAMING_SNAKE_CASE` must still be used even with the lower case `e_` prefix.
 
-Enumerator fields must also be `SCREAMING_SNAKE_CASE` and use the enumerator
-name as a prefix.
-
 ```pawn
-static enum E_PLAYER {
-    E_PLAYER_CASH,
-    Float:E_PLAYER_HEALTH,
+static enum E_PLAYER_DATA {
+    e_PlayerCash,
+    Float:e_PlayerHealth,
```

Comment:
They should be `e_PLAYER_CASH` and `e_PLAYER_HEALTH`, going by both convention and what the text immediately before says.

Reply:
Fixed the text in the latest commit